sections/introduction.include

```diff
@@ -697,9 +697,11 @@
       There are many constructs that can be used to try to trick a site into executing code. Here
       are some that authors are encouraged to consider when writing safelist filters:
 
-      * When allowing harmless-seeming elements like <{img}>, it is important to safelist any
-        provided attributes as well. If one allowed all attributes then an attacker could, for
-        instance, use the <a attribute for="GlobalEventHandlers"><code>onload</code></a> attribute to run arbitrary script.
+      * When allowing harmless-seeming elements like <{img}>, it is important to safelist only the
+        necessary attributes (that are needed for this specific demand). If one allowed all
+        attributes then an attacker could, for instance, use the
```

Comment:
How about: 

[...] <{img}>, exercise the principle of least-privilege and limit the element's attributes to only those that are needed (e.g., via a safelist). [...]